src/main/scala/org/broadinstitute/dsde/firecloud/webservice/BillingApiService.scala

```diff
@@ -1,86 +1,23 @@
 
-trait BillingApiService extends FireCloudDirectives {
-  private val billingUrl = FireCloudConfig.Rawls.authUrl + "/billing"
+trait BillingApiService extends FireCloudDirectives with StreamingPassthrough {
   private val userBillingUrl = FireCloudConfig.Rawls.authUrl + "/user/billing"
-  private val v2BillingUrl = FireCloudConfig.Rawls.authUrl + "/billing/v2"
 
   val billingServiceRoutes: Route =
     pathPrefix("billing") {
-      pathEnd {
-        post {
-          passthrough(billingUrl, POST)
-        }
-      } ~
-        pathPrefix(Segment) { projectId =>
-          path("members") {
-            get {
-              passthrough(s"$billingUrl/$projectId/members", GET)
-            }
-          } ~
-            // workbench project role: owner or user
-            path(Segment / Segment) { (workbenchRole, email) =>
-              (delete | put) {
-                extract(_.request.method) { method =>
-                  passthrough(s"$billingUrl/$projectId/$workbenchRole/$email", method)
-                }
-              }
-            }
-        } ~
-        pathPrefix("v2") {
-          pathEnd {
-            post {
-              passthrough(v2BillingUrl, POST)
-            }
-          } ~
-            pathPrefix(Segment) { projectId =>
-              pathEnd {
-                (get | delete) {
-                  extract(_.request.method) { method =>
-                    passthrough(s"$v2BillingUrl/$projectId", method)
-                  }
-                }
-              } ~
-                pathPrefix("members") {
-                  pathEnd {
-                    get {
-                      passthrough(s"$v2BillingUrl/$projectId/members", GET)
-                    }
-                  } ~
-                    // workbench project role: owner or user
-                    path(Segment / Segment) { (workbenchRole, email) =>
-                      (delete | put) {
-                        extract(_.request.method) { method =>
-                          passthrough(s"$v2BillingUrl/$projectId/members/$workbenchRole/$email", method)
-                        }
-                      }
-                    }
-                } ~
-                path("billingAccount") {
-                  (put | delete) {
-                    extract(_.request.method) { method =>
-                      passthrough(s"$v2BillingUrl/$projectId/billingAccount", method)
-                    }
-                  }
-                } ~
-                path("spendReportConfiguration") {
-                  (get | put | delete) {
-                    extract(_.request.method) { method =>
-                      passthrough(s"$v2BillingUrl/$projectId/spendReportConfiguration", method)
-                    }
-                  }
-                }
-            }
-        }
+      // all paths under /api/billing pass through to the same path in Rawls
+      streamingPassthrough(Uri.Path("/api/billing") -> Uri(FireCloudConfig.Rawls.authUrl + "/billing"))
```

Comment:
I'm not grokking this -- will the /billing/v2 endpoints deleted above still be passed through with this approach?

Reply:
yes - consider this as a wildcard; it will pass `${orchestration}/api/billing*` to `${rawls}/api/billing*`.

Reply:
@aherbst-broad does this work for you?

Reply:
Yes, looks good.